src/containers/Accounts/AccountTransactionsTable/index.js

```diff
@@ -64,43 +64,42 @@ export const AccountTxTable = (props) => {
     return (
       <li
         key={tx.hash}
-        className={`transaction-li tx-type ${tx.type} ${
+        className={`transaction-li anchor-mask tx-type ${tx.type} ${
           success ? 'success' : 'fail'
         }`}
       >
-        <Link to={`/transactions/${tx.hash}`}>
-          <div className="upper">
-            <div className="col-account">
-              <div className="transaction-address" title={tx.account}>
-                {tx.account}
-              </div>
+        <Link to={`/transactions/${tx.hash}`} className="mask-overlay" />
```

Comment:
All of this was just moving the content out of the `<Link>` component and adding `className="mask-overlay"` to the `<Link>`